app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
Interesting, I'm not sure if I understand what Rubocop was complaining about. Adding an argument does nothing to the lines of code or the complexity of the method. Are you sure you didn't refactor beyond this?